_scripts/test_linux.sh

```diff
@@ -21,7 +21,10 @@ function getgo {
 }
 
 if [ "$version" = "gotip" ]; then
-	exit 0
+	# TODO: remove this
```

Comment:
This will be removed once the regabi stuff is a little less chaotic I assume?

Reply:
Yes, I'm thinking of doing it on the timeline we'd normally support the next version of Go, remove this and fix the test failures.